server/lib/notifications/email.ts

```diff
@@ -413,15 +414,21 @@ export const notifyByEmail = async (activity: Activity) => {
         }
         break;
       }
-      await notify.collective(activity, { collectiveId: activity.data.collective.id });
+      const HostCollectiveId = await collective.getHostCollectiveId();
+      await notify.collective(activity, {
+        collectiveId: activity.data.collective.id,
+        cc: (HostCollectiveId) ? HostCollectiveId : 'no-reply@opencollective.com',
```

Comment:
This part is the core problem and the reason why there's no `cc` currently: we need a way for host admins to define a custom email address for the host account. See https://github.com/opencollective/opencollective/issues/2263